releases/README.md

```diff
@@ -42,21 +42,27 @@
 3. Push tag and Github release
 
    1. Push the tag to `git@github.com:containerd/containerd.git`.
-      NOTE: this will kick off CI building of the release binaries.
+      NOTE: this will kick off CI to prepare the Github release and binaries.
 
-   2. Create the Github release using the `Tag version` which was just pushed. Use the first
-      line outputted from the release tool as the `Release title` and the remainder of the
-      output for the description. No alteration of the release output should be needed.
-      Ensure `pre-release` is checked if an `-rc`.
-      NOTE: This should be done immediately after pushing the tag, otherwise CI may create the release
-      when the binaries are pushed.
+   2. Verify release job has started in the actions tab and wait for release job to complete
+      successfully. If the job fails, attempt to restart it. If the job continuously fails,
+      delete the tag from Github as soon as possible and fix the issue before restarting the
```

Comment:
```suggestion
      delete the tag from GitHub as soon as possible and fix the issue before restarting the
```

Reply:
I don't think we should delete the tag; once it's tagged, it's done; go module proxies may already have it stored, and will mark the module as invalid.

The correct step here would be to do a new release and retract the old one.

The alternative would be to tag at the end of the release (after validating). I know we do so for moby (once we're happy with a build, the commit is "blessed" by tagging it). But I guess that's a more extensive change in process

Reply:
Yeah, that would be a bit more involved. When there is changes to the process or something large, we can test the release process on a fork. Deleting a tag is not ideal but it is the best solution if the tag is bad for some reason, the caching can get resolved.